packages/horizon/contracts/interfaces/ITAPCollector.sol

```diff
@@ -25,6 +25,8 @@ interface ITAPCollector is IPaymentsCollector {
 
     /// @notice The Receipt Aggregate Voucher (RAV) struct
     struct ReceiptAggregateVoucher {
+        // The ID of the payment
+        address paymentId;
```

Comment:
Thinking about it a bit more I think the term `collectionId` is a bit more representative of what this is. For each data service these are "collection buckets", wdyt?